ratpack-kafka-producer/src/main/java/smartthings/ratpack/kafka/KafkaProducerModule.java

```diff
@@ -73,5 +80,53 @@ public boolean isEnabled() {
 		public void setEnabled(boolean enabled) {
 			this.enabled = enabled;
 		}
+
+		public void setLingers_ms(Optional<Long> lingers_ms) {
```

Comment:
This is due to my comments here https://github.com/SmartThingsOSS/ratpack-kafka/pull/15 . Mainly I want to only set the properties to kafka if values are set, and not maintain all of our own defaults that mirror the driver.  Having the set be optional seems a little odd to me as well.